apps/researcher/.env.development

```diff
@@ -5,3 +5,5 @@
 SEARCH_PLATFORM_ELASTIC_ENDPOINT_URL=https://api.colonial-heritage.triply.cc/datasets/testing/researcher-search-v1/services/researcher-search-v1-es/elasticsearch
 SEARCH_PLATFORM_SPARQL_ENDPOINT_URL=https://api.colonial-heritage.triply.cc/datasets/testing/researcher-search-v1/services/researcher-search-v1-sparql/sparql
 KG_SPARQL_ENDPOINT_URL=https://api.colonial-heritage.triply.cc/datasets/testing/researcher-entities-v1/services/researcher-entities-v1-sparql/sparql
+
+DATASET_BROWSER_URL="https://dev.datasets.colonialcollections.nl"
```

Comment:
For consistency with the other vars: remove the quotes and the newline on line 8?